src/services/UserService.js

```diff
@@ -0,0 +1,23 @@
+import { api } from '@/config/api';
+let response = []
+
+export async function signUp(data) {
+    response = await api.post('/users', data).then(res => {
```

Comment:
Here you combine 2 ways of handling asynchronous requests, on the one hand with async/await and on the other with promises.

Please use async/await or promises